internal/service/advancedclustertpf/responses.go

```diff
@@ -52,3 +54,22 @@ func ReadClusterProcessArgsResponse() (*admin.ClusterDescriptionProcessArgs20240
 	err := json.Unmarshal([]byte(responseJSON), &SDKModel)
 	return &SDKModel, err
 }
+
+func StoreCreatePayload(payload *admin.ClusterDescription20240805) error {
```

Comment:
Similar to the rest of this file, this will be removed when real SDK is used and we can use mocking instead